base/cvd/cuttlefish/host/commands/cvd/selector/instance_group_record.cpp

```diff
@@ -92,11 +92,21 @@ void LocalInstanceGroup::SetProductOutPath(
   group_proto_.set_product_out_path(product_out_path);
 }
 
+bool LocalInstanceGroup::InstanceIsActive(const cvd::Instance& instance) {
+  switch (instance.state()) {
+    case cvd::INSTANCE_STATE_RUNNING:
+    case cvd::INSTANCE_STATE_STARTING:
+    case cvd::INSTANCE_STATE_STOPPING:
+    case cvd::INSTANCE_STATE_PREPARING:
+    case cvd::INSTANCE_STATE_UNREACHABLE:
+      return true;
+    default:
```

Comment:
Rather than using `default` to cover remaining cases, can this exhaustively list all the `enum` members?

See [abseil.io/tips/147](https://abseil.io/tips/147).

Reply:
Done

Reply:
Sorry, I missed part of the text in the resource I linked.

> Exhaustive switch statements on proto3 enum types are not recommended. The parser doesn’t guarantee that enum fields will have enumerator values. Additionally, it’s not possible to write an exhaustive `switch` statement over proto3 enum types without referencing special sentinel enumerators that should be considered internal implementation details of the protobuf tools.

So the last group should be `default: LOG(FATAL)` in that case?